easybuild/easyconfigs/t/TensorFlow-Datasets/TensorFlow-Datasets-4.7.0-foss-2021a-CUDA-11.3.1.eb

```
@@ -0,0 +1,55 @@
+easyblock = 'PythonBundle'
+
+name = 'TensorFlow-Datasets'
+version = '4.7.0'
+versionsuffix = '-CUDA-%(cudaver)s'
+
+homepage = 'https://www.tensorflow.org/datasets'
+description = """
+TensorFlow Datasets is a collection of datasets ready to use, with TensorFlow or other Python ML frameworks, such as
+Jax. All datasets are exposed as tf.data.Datasets , enabling easy-to-use and high-performance input pipelines.
+"""
+
+toolchain = {'name': 'foss', 'version': '2021a'}
+
+dependencies = [
+    ('CUDA', '11.3.1', '', SYSTEM),
+    ('Python', '3.9.5'),
+    ('tqdm', '4.61.2'),
+    ('TensorFlow', '2.6.0', '-CUDA-%(cudaver)s'),
+]
+
+sanity_pip_check = True
+use_pip = True
+
+exts_list = [
+    ('promise', '2.3', {
+        'checksums': ['dfd18337c523ba4b6a58801c164c1904a9d4d1b1747c7d5dbf45b693a49d93d0'],
+    }),
+    ('googleapis-common-protos', '1.56.4', {
+        'modulename': 'google',
+        'checksums': ['c25873c47279387cfdcbdafa36149887901d36202cb645a0e4f29686bf6e4417'],
+    }),
+    ('tensorflow-metadata', '1.10.0', {
+        'source_tmpl': '%(name)s-%(version)s-py3-none-any.whl',
+        'source_urls': [(
+            'https://files.pythonhosted.org/packages/3a/86/'
+            + '2b3251bb560068f31817d9b678588098e28f396c1f6b88c57cf5d28670af/'
+            + 'tensorflow_metadata-1.10.0-py3-none-any.whl'
+            + '#sha256=e3ff528496105c0d73b2a402877525b1695635378fbe5c1b47ac7b3780816bb3'
+        )],
+        'checksums': ['e3ff528496105c0d73b2a402877525b1695635378fbe5c1b47ac7b3780816bb3'],
+    }),
+    ('etils', '0.8.0', {
+        'checksums': ['d1d5af7bd9c784a273c4e1eccfaa8feaca5e0481a08717b5313fa231da22a903'],
+    }),
+    (name, version, {
+        'download_filename': 'v(%version)s.tar.gz',
+        'filename': '%(name)s-v(%version)s.tar.gz',
+        'source_tmpl': 'v%(version)s.tar.gz',
+        'source_urls': ['https://github.com/tensorflow/datasets/archive'],
+        'checksums': ['ed7c3b959d10ba762137e18b93631ab42f4ed0915bc3d9ce98ee66eef9c61418'],
```

can we use the version on PYPI instead?
```suggestion
        'source_tmpl': SOURCELOWER_TAR_GZ,
        'checksums': ['590faf3763bc14757906b36c718389eeded533ee8ae4d030ede140db86aca4cd'],
```